feature(docker): load modules on docker (#540)
emqx/emqx-rel: deploy/docker/README.md

``docker run -d --name emqx -e EMQX_LISTENER__TCP__EXTERNAL=1883 -p 18083:18083 -p 1883:1883 emqx/emqx:latest``

#### EMQ Loaded Modules Configuration

| Oprtions                 | Default            | Description                           |
| ------------------------ | ------------------ | ------------------------------------- |
| EMQX_LOADED_MODULES       | see content below  | default modules emqx loaded            |

Default environment variable ``EMQX_LOADED_MODULES``, including

- ``emqx_mod_acl_internal``
- ``emqx_mod_presence``

```bash
# The default EMQX_LOADED_MODULES env
EMQX_LOADED_MODULES="emqx_mod_acl_internal,emqx_mod_acl_internal"
```

> For example, load ``emqx_mod_delayed`` and ``emqx_mod_rewrite`` modules, set it into ``EMQX_LOADED_MODULES`` and use any separator to separates it.

Reviewer: ```suggestion
For example, set ``EMQX_LOADED_MODULES=emqx_mod_delayed,emqx_mod_rewrite`` to load these two plugins.
```
When giving example, as a user, I'd prefer to have something which can copy-past

You can use comma, space or other separator that you want.

> All the plugin you defined in env ``EMQX_LOADED_PLUGINS`` will be loaded.

Reviewer: ```suggestion
All the plugin you defined in env ``EMQX_LOADED_MODULES`` will be loaded.
```

```bash
EMQX_LOADED_MODULES="emqx_mod_delayed,emqx_mod_rewrite"
EMQX_LOADED_MODULES="emqx_mod_delayed emqx_mod_rewrite"

> EMQX_LOADED_MODULES="emqx_mod_delayed | emqx_mod_rewrite"

Reviewer: missing the closing triple-backslash quote

# The default EMQX_LOADED_PLUGINS env
EMQX_LOADED_PLUGINS="emqx_recon,emqx_retainer,emqx_management,emqx_dashboard"
```

> For example, load ``emqx_auth_redis`` and ``emqx_auth_mysql`` plugin, set it into ``EMQX_LOADED_PLUGINS`` and use any separator to separates it.

Reviewer: ```suggestion
For example, set it into ``EMQX_LOADED_PLUGINS= emqx_auth_redis,emqx_auth_mysql`` to load these two plugins.
```

We have the delimiters explained above and below, do not need to repeat

You can use comma, space or other separator that you want.

> All the plugin you defined in env ``EMQX_LOADED_PLUGINS`` will be loaded.

Reviewer: ```suggestion
All plugins defined in ``EMQX_LOADED_PLUGINS`` will be loaded.
```

docker run -d --name emqx -p 18083:18083 -p 1883:1883 -p 4369:4369 \
    -e EMQX_LISTENER__TCP__EXTERNAL=1883 \

>     -e EMQX_LOADED_PLUGINS="emqx_auth_redis" \

Reviewer: just a q: why delete ?

Author: > just a q: why delete ?

Previously, when users used `EMQX_LOADED_PLUGINS` to load plug-ins, they must manually specify `emqx_recon, emqx_retainer, emqx_management, emqx_dashboard`, otherwise these plug-ins cannot start normally and will cause emqx to fail to start. 

Now regardless of whether the user specifies these plug-ins, these plug-ins will work normally start up
